src/components/SummaryCard/SummaryCardAction/__tests__/SummaryCardAction.spec.js

```diff
@@ -0,0 +1,113 @@
+/**
+ * @file Summary card tests.
+ * @copyright IBM Security 2020
+ */
+
+import { render } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import React from 'react';
+
+import { SummaryCardAction } from '../../../..';
+
+describe('SummaryCardAction', () => {
```

Comment:
Will we need tests for `hasIconOnly` and `onClick` also?

Reply:
Those are `Button` props, passed through to the inner `Button`... 😅 
The only thing we'd really be checking with `hasIconOnly` is if the boolean is doing its thing?

With `onClick`, what specifically would you be testing for / what do you feel is missing here?

Reply:
Because these are consumer-facing props, if they were removed, we wouldn't know about it

Reply:
I'm not sure what you mean here -- if the prop is removed on our end, then we would know about it because we removed it? 🤷‍♀ 

But I can add a test to move this along. Not exactly sure what I am able to check, since we don't control the icon here, but I'll work something out.

Reply:
We may know about it, or we may not if we don't catch it! 😆The unit tests are in place to protect the functionality a component is expected to provide in case we introduce a change that breaks it.